client/blocks/inline-help/contextual-help.js

```diff
@@ -931,6 +931,31 @@ const contextLinksForSection = {
 			),
 		},
 	],
+	checkout: [
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/plan-features/' ),
+			post_id: 140323,
+			title: translate( 'WordPress.com Plan Details' ),
+			description: translate( "We'll help you pick the best fit for your needs and goals." ),
+		},
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/manage-purchases/#refund-policy' ),
```

Comment:
It's too bad it doesn't jump down to this section on the page.

Reply:
Yeah, agree. I can see if it'd be possible to scrollTo the element.